package.json

```diff
@@ -80,7 +81,7 @@
     "rxjs": "^5.5.2",
     "systemjs": "^0.20.19",
     "tslint": "^5.8.0",
-    "typescript": "^3.1.6",
+    "typescript": "^2.9.2",
```

Comment:
@strausr , why did we decrease version?

Reply:
This was the original version, the one that is compatible with our Angular version